review/templates/review/course.html

```diff
@@ -53,6 +54,13 @@
     <tr class="{% if exercise.paused %}paused {% endif %}">
         <td><a href="{% url 'exercise' course_key=course.key exercise_key=exercise.key %}">{{ exercise.name }}</a></td>
         <td>{{ exercise.submissions.count }}</td>
+        {% if exercise.submissions.count %}
+        <td class="warning">
+          <strong>{{ exercise.submissions.count }}</strong>
+        </td>
+        {% else %}
+        <td>{{ exercise.submissions.count }}</td>
+        {% endif %}
```

Comment:
Problematic submissions equals the number of all submissions? This must be a placeholder value. Implementation is missing.